content/en/testing-guide/_index.adoc

```diff
@@ -0,0 +1,62 @@
+
+---
+title: "PAPRa Testing Guide (13 March 2021 Edition)"
+linkTitle: "PAPRa Testing Guide"
+weight: 1
+description: >
+  How we tested the PAPRa to arrive at claims of N95/N99 levels of protection
+---
+
+== Quick Synopsis
+
+This page will walk you through how we established that the PAPRa can offer N95 levels of protection on a mask when worn by someone with a beard, and N99 levels of protection by the fan box itself.  These tests are by no means exhaustive; far more testing is needed to ensure that these results can be achieved consistently, and the testing should be done in a more controlled environment than what we have done.
+
+Furthermore, we offer no warranty, explicit or implied, that any device that you create will achieve similar standards of filtration.  We are currently working to determine the best path forward for individuals who make their PAPRs to test them.
+
+== Parts Needed
+
+In addition to the PAPR to be tested, we tested using a https://tsi.com/products/respirator-fit-testers/portacount-respirator-fit-tester-8038/[Portacount] with a hose assembly as depicted below:
+
+[link=https://photos.smugmug.com/Tetra-Testing/PAPRa-Build-13-March-2021/i-N5dwqjg/0/a18cf5f4/5K/_MMR0431-5K.jpg]
+image::https://photos.smugmug.com/Tetra-Testing/PAPRa-Build-13-March-2021/i-N5dwqjg/0/a18cf5f4/L/_MMR0431-L.jpg[] 
+
+It is important to note that the Portacount is effective at showing the isolation of airborne particles, but not at identifying the sizes of particles measured, and did not have a setting for an N100 level of protection.  When testing for N95 levels of protection, the Portacount has a range from 0 to 200+, with 100 passing, while N99 testing has a range from 0 to 2000+, with 500 passing.  All tests depicted in this document were done using online testing, waiting several seconds to ensure that the readings were stable before recording them.
```

Comment:
I'd also convert this text block to an [admonition](https://docs.asciidoctor.org/asciidoc/latest/blocks/admonitions/), prepended with the **NOTE** keyword